Update lambda function to use python3.9

pjochem4 opened the pull request:
I made a venv with Python3.9 and changed lambda-publish.sh to use the python3.9 packages.
If my understanding is correct then when I run lambda-publish.sh, AWS will update our lambda function to use the new Python code. In the past they upgraded from Python2.7 to Python3.6. The PRs for that are [here](https://github.com/path-robotics/s3apt/commit/1419f643c7b3083be2992dfbe2a0668a44d99a4d) and [here](https://github.com/path-robotics/s3apt/commit/ace6a8ac83cedf0ee453da4503b23418a0e2399a). More info about creating a lambda function with a virtual environment is [here](https://docs.aws.amazon.com/lambda/latest/dg/python-package.html#python-package-update-venv). 

I am going to wait until this is merged to run the upload script. Don't want to be that guy who breaks other people's playbook runs :sweat_smile: 

Below is how I setup the environment on my machine. 

1. Install Python3.9 and its venv
```sudo apt install software-properties-common
sudo add-apt-repository ppa:deadsnakes/ppa
sudo apt install python3.9
sudo apt install python3.9-venv
```
2. Create Python3.9 venv
```
python3.9 -m venv ./venv
```
3. Setup config.py with the APT_REPO_BUCKET_NAME = "path-apt-repo"
4. Activate the Python venv ```source venv/bin/activate```
5. Install packages into the virtual environment ```pip install -r requirements.txt```
6. Deactivate venv ```deactivate```

pjochem4 committed:
Update lambda function to use python3.9

gsheeleypath commented on lambda-publish.sh:
```diff
@@ -4,6 +4,6 @@
 zip  code.zip s3apt.py config.py
 
-(cd venv/lib/python2.7/site-packages/ ; zip -r ../../../../code.zip *)
+(cd venv/lib/python3.9/site-packages/ ; zip -r ../../../../code.zip *)
```
I would update the README as well.

pjochem4 committed:
Update lambda function name

Looks like the name of the lambda changed. This
commit changes it to match what is in AWS lambda

pjochem4 committed:
Update Python version in the README
